phat/example-oracles/lens_stats/src/index.ts

```diff
@@ -0,0 +1,169 @@
+import "@phala/pink-env";
```

Comment:
It makes sense to make this a template so devs can easily refer to this to define their oracle logic.
Some more docs will be needed like what pink runtime functions can be used, and what is not supported (async, .e.t.c.).

Reply:
Yes, we should enhance the docs to phat-quickjs.

Reply:
A brief introduction has been added to the `README.md`. To the docs for the JS ability, it should out of the scope of this PR. Let's do it in the phat-quickjs repo.